lib/utils/file_utils.dart

```diff
@@ -25,31 +25,17 @@ class FileUtils {
     String? fileTransferId,
     Function()? onComplete,
   }) async {
-    await showModalBottomSheet(
-      context: NavService.navKey.currentContext!,
-      backgroundColor: Colors.white,
-      builder: (context) => EditBottomSheet(
-        onConfirmation: () async {
-          var file = File(filePath);
-          if (await file.exists()) {
-            await file.delete();
-          }
-          if (fileTransferId != null) {
-            await Provider.of<MyFilesProvider>(
```

Comment:
why did we remove this ? 
This function deletes the file from the local provider

Reply:
@Sardar1208 When I do testing, there are no problems have shown up yet. I will add them back for sure

Reply:
Okay sure